locale/zh/messages.json

```diff
@@ -8271,7 +8271,7 @@
     ]
   },
   "Rejected offer analysis is not currently supported for {frameworkName}.": {
-    "translation": "目前不支持已拒绝 offer 分析 {frameworkName}。",
```

Comment:
There are several translations lost in this PR 🤔

Reply:
I know, I have to update it, but since it is not in the sprint, it's low priority right now. Still, thanks for the info.